frontend/src/app/choreography/choreography.component.ts

```diff
@@ -232,4 +216,38 @@ export class ChoreographyComponent implements OnInit {
   removePersonFromCarpet(index: number): void {
     this.clearItem(this.choreography.frames[this.activeFrame].subframes[this.activeSubframe].grid[index]);
   }
+
+  clearItem(item: ChoreographyItem): void {
+    item.text = null;
+    item.color = '';
+    item.position = ['center', 'center'];
+    item.sign = { text: '', color: '' };
+  }
+
+  switchGroupType(groupType: GroupType): void {
+    this.activeChoreographyItem!.text = createEmptyGroup(groupType);
+    this.activeGroupType = this.activeChoreographyItem!.text.type;
```

Comment:
**question:** can `activeGroupType` be calculated based on `this.activeChoreographyItem` like `isActiveItemGroup`?